### ISSUE TITLE
Query Params not working in SSR but do in browser

### ISSUE THREAD

**Issue by MarcGodard** [canjs/feathers-query-logic#4]
When executing the following, responses vary from SSR to browser:

Works the same in both:
```
Model.getList({}).then(data => {
     console.log(data)
})
```
```
Model.getList({ $skip: 0, $limit: 10 }).then(data => {
     console.log(data)
})
```

Does not work the same in both:
```
Model.getList({ active: true }).then(data => {
     console.log(data)
})
```
```
Model.getList({ active: true, $skip: 0, $limit: 10 }).then(data => {
     console.log(data)
})
```

SSR returns no data (console.log in terminal) yet the browser (after detaching) shows correct data (console.log in browser console).

The API does receive the query parameters from both calls.

**Comment by chasenlehara** [comment]
Gitter convo for context: https://gitter.im/donejs/donejs?at=5c2520c4db5b5c6883169d37

**Comment by MarcGodard** [comment]
This really might be a bigger problem as I cannot see it being this repo's code doing it.

**Comment by MarcGodard** [comment]
After investigating this more, it looks like it is an issue with the boolean.

I changes it to the following (variable is in the model):
```
Model.getList({ variable: 'string' }).then(data => {
     console.log(data)
})
```
This one worked in client and SSR.

Then I tested without variable being in the model, and it didn't work in SSR but worked in the client.

I worked on a fork of this code, and this code isn't the issue. It is treating booleans differently.

In the model I tried:
```
  variable: 'boolean',
  another: QueryLogic.makeEnum([true, false]),
```
To see if it would make a difference, it didn't, both types of variables failed. I also tried to use 'true' as a string, and that also failed. Possibly because it didn't match the model.
